exchanges/btcmarkets/btcmarkets_websocket.go

```diff
@@ -468,10 +468,7 @@ func checksum(ob *orderbook.Base, checksum uint32) error {
 
 // concat concatenates price and amounts together for checksum processing
 func concat(liquidity orderbook.Tranches) string {
-	length := 10
-	if len(liquidity) < 10 {
-		length = len(liquidity)
-	}
+	length := min(10, len(liquidity))
 	var c string
 	for x := range length {
```

Comment:
🐒 
* Since we're not re-assigning to it anymore, don't think we necessarily need a var:
```suggestion
	var c string
	for x := range 	min(10, len(liquidity)) {
```
* c can move to sig if you want:
```suggestion
func concat(liquidity orderbook.Tranches) (c string) {
	for i := range min(10, len(liquidity)) {
		c += trim(liquidity[i].Price) + trim(liquidity[i].Amount)
	}
	return c
```
* 🥃 And since we've come this far, `concat` is a bit of a loose name given it's in the whole of the exchange package namespace. `concatOrderbookLiquidity` or something